packages/fast-layouts-react/src/utilities/breakpoint-tracker.ts

```diff
@@ -56,7 +62,27 @@ class BreakpointTracker {
     public set breakpoints(breakpointConfig: Breakpoints) {
         this._breakpoints = breakpointConfig;
 
-        this.breakpoint = identifyBreakpoint(window.innerWidth, this._breakpoints);
+        this.breakpoint = canUseDOM()
+            ? identifyBreakpoint(window.innerWidth, this._breakpoints)
+            : this.defaultBreakpoint;
+    }
+
+    /**
+     * Gets the default breakpoint value
+     */
+    public get defaultBreakpoint(): Breakpoint {
+        return this._defaultBreakpoint;
+    }
+
+    /**
+     * Sets the default breakpoint value
+     */
+    public set defaultBreakpoint(breakpoint: Breakpoint) {
+        this._defaultBreakpoint = breakpoint;
+
+        if (!canUseDOM()) {
+            this.breakpoint = this.defaultBreakpoint;
```

Comment:
should this call update() instead here?

Reply:
likewise with `set breakpoints()` function. these both update `this.breakpoint` but do not trigger any `notifySubscribers`. Meaning even if `update()` is called notifySubscribers _**still**_ won't be called which feel wrong.

Reply:
Good question - my expectation would be if current breakpoint *changes* then update would get called. Theoretically you could change the default breakpoint and breakpoints array without the active breakpoint changing.

Reply:
`update()` already contains the check `if (this.breakpoint !== breakpoint) { //do work }`
To reduce the redundancy of calculating the new breakpoint I'll update to calling `update()` here and in `set breakpoints`

Reply:
Yea I think adding to both is a good call. Thanks @JamesBurnside